frontend/simple/utils/lazyLoadVue.js

```diff
@@ -0,0 +1,17 @@
+// use VueScript2 to handle lazy-loading of routes in conjunction
+// with browserify-shim. See Gruntfile for details.
+import Vue from 'vue'
+import VS2 from 'vue-script2'
+// import VS2 from '../../../ignored/Script2'
+Vue.use(VS2)
+
+export default function lazyLoadVue (component: string, base: string = '/simple/js') {
+  // if we wanted to support vue-hot-reload-api in
+  // lazy-loaded modules, then we could do something like this in main.js:
+  // window.HOTAPI = require('vue-hot-reload-api')
+  return function (resolve: Function, reject: Function) {
+    VS2.load(`${base}/${component}.js`)
+      .then(() => resolve(window[component]))
+      .catch((err) => reject(err))
+  }
+}
```

Comment:
This file doesn't make sense to me. It seems to be only about one thing, a single function, and yet it does two things (setup vue-script2 as well).

In summary, two issues:

1. I think it is a bad idea to start doing 1-file-per-function.
2. This file is poorly named as what it does / what it's for isn't adequately described by the filename

Reply:
So, I can keep it as it was before, `lazyLoadVue` and `mapValues` on the same file, and we can see later how to organize it better on another PR.

Reply:
Regarding travis failing, I was at the train with low wifi, so I believe the push didn't successed. But it's fixed now :)